### DISCUSSION TITLE
是否可以给番剧做一个单集收藏甚至每一集可以设置自己的空降点？

### DISCUSSION THREAD

**Discussion by KaiTang2** [Richasy/Bili.Uwp#580] [想法]
有时候一整个番剧看完并不是所有集都会记住，拿柯南举例，1000多集，非常想要个单集收藏的功能，可以快速找到我想看的那集，或者某集中精彩的部分直接空降过去，每次都慢慢找太费时间了

**Comment by Q-Tai-mu** [comment]
想法很棒(๑•̀ㅂ•́)و✧

**Reply by KaiTang2** [reply]
实际执行起来呢？

**Reply by Q-Tai-mu** [reply]
太麻烦了，这个功能相对于收藏+历史播放位置，而对应的集数是N，做起来很耗时间
